server/tests/repositoryTests/ratings.test.js

```diff
@@ -0,0 +1,10 @@
+import chai from 'chai';
+import ratingRepo from '../../repository/ratingRepository';
+
+const { expect } = chai;
+
+describe('Ratings:', () => {
+  it('should be a function', () => {
+    expect(ratingRepo.createRating).to.be.a('function');
```

Comment:
I have more tests on the `rating` functionality in the article.test.js file since ratings pertain to articles. I created this to have at least one test on the rating repo